apps/api/v1/test/lib/bookings/_get.integration-test.ts

```diff
@@ -105,4 +105,89 @@ describe("GET /api/bookings", async () => {
     expect(responseData.bookings.find((b) => b.id === proUserBooking.id)).toBeUndefined();
     expect(groupedUsers.size).toBeGreaterThanOrEqual(2);
   });
+
+  // Tests for upcoming bookings feature - START
```

Comment:
Instead of using `// Tests for upcoming bookings feature - START` and // Tests for upcoming bookings feature - END  it is better to group all of these `it` tests into a `describe` block:

```
describe("test upcoming bookings filter", () => {
    it("Returns only upcoming bookings when upcoming=true for regular user", async () => {
        ...
    });
    ...
});
```